docs/clients/go-client-api.md

```diff
@@ -0,0 +1,337 @@
+---
+title: Try Two Types of APIs
+summary: Learn how to use the Raw Key-Value API and the Transactional Key-Value API in TiKV.
+category: user guide
+---
+
+# Try Two Types of APIs
+
+To apply to different scenarios, TiKV provides [two types of APIs](../overview.md#two-types-of-apis) for developers: the Raw Key-Value API and the Transactional Key-Value API. This document uses two examples to guide you through how to use the two APIs in TiKV. The usage examples are based on multiple nodes for testing. You can also quickly try the two types of APIs on a single machine.
+
+## Try the Raw Key-Value API
+
+To use the Raw Key-Value API in applications developed by golang, take the following steps:
+
+1. Install the necessary packages.
+
+    ```bash
+    go get -v -u github.com/pingcap/tidb/store/tikv
+    ```
+
+2. Import the dependency packages.
+
+    ```go
+    import (
+        "fmt"
+        "github.com/pingcap/tidb/config"
+        "github.com/pingcap/tidb/store/tikv"
+    )
+    ```
+
+3. Create a Raw Key-Value client.
+
+    ```go
+    cli, err := tikv.NewRawKVClient([]string{"192.168.199.113:2379"}, config.Security{})
+    ```
+
+    Description of two parameters in the above command:
+
+    - `string`: a list of PD servers’ addresses
+    - `config.Security`: used for establishing TLS connections, usually left empty when you do not need TLS
```

Comment:
Is it expected that the user will understand how to set up TLS here if they want it?

Reply:
Yes.